c++/test/Makefile.in

```diff
@@ -1097,7 +1117,21 @@ clean-am: clean-checkPROGRAMS clean-generic clean-libtool \
 	mostlyclean-am
 
 distclean: distclean-am
-	-rm -rf ./$(DEPDIR)
+		-rm -f ./$(DEPDIR)/dsets.Po
+	-rm -f ./$(DEPDIR)/h5cpputil.Po
+	-rm -f ./$(DEPDIR)/tarray.Po
+	-rm -f ./$(DEPDIR)/tattr.Po
+	-rm -f ./$(DEPDIR)/tcompound.Po
+	-rm -f ./$(DEPDIR)/tdspl.Po
+	-rm -f ./$(DEPDIR)/testhdf5.Po
+	-rm -f ./$(DEPDIR)/tfile.Po
+	-rm -f ./$(DEPDIR)/tfilter.Po
+	-rm -f ./$(DEPDIR)/th5s.Po
+	-rm -f ./$(DEPDIR)/tlinks.Po
+	-rm -f ./$(DEPDIR)/tobject.Po
+	-rm -f ./$(DEPDIR)/trefer.Po
+	-rm -f ./$(DEPDIR)/ttypes.Po
+	-rm -f ./$(DEPDIR)/tvlstr.Po
```

Comment:
I'm just curious, why are these listed individual instead of using wildcard?  Should I do something differently with the files?